base_delivery_carrier_files_document/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2012 Camptocamp SA
+# Copyright 2021 Sunflower IT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Store carrier files as attachments",
+    "version": "14.0.1.0.0",
+    "category": "Generic Modules/Warehouse",
+    "summary": """
+    Allow to store carrier files as attachments.
+    Auto-install when the module Document and
+    Base Delivery Carrier Files are installed.
+    """,
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "Camptocamp, Sunflower IT, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["base_delivery_carrier_files", "attachment_indexation"],
```

Comment:
change MR to indicate it depends on https://github.com/OCA/delivery-carrier/pull/363

Reply:
So they already merged this to the main OCA v14.0 repo